newa/cli.py

```diff
@@ -155,6 +165,34 @@ def cmd_event(ctx: CLIContext, errata_ids: list[str], compose_ids: list[str]) ->
     if not errata_ids and not compose_ids:
         raise Exception('Missing event IDs!')
 
+    def apply_mapping(string: str,
+                      mapping: Optional[list[str]] = None,
+                      regexp: bool = True) -> str:
+        # define default mapping
+        if not mapping:
+            mapping = [
+                '\\.GA$=',
```

Comment:
raw strings instead? No need to escape that much

Reply:
Good point!

Reply:
Updated